ports/psoc6/modules/network/network_ifx_wcm.c

```diff
@@ -740,46 +725,46 @@ STATIC mp_obj_t network_ifx_whd_config(size_t n_args, const mp_obj_t *args, mp_m
             case MP_QSTR_antenna: {
                 uint8_t buf[4];
                 uint32_t ret = whd_wifi_get_ioctl_buffer(whd_ifs[self->itf], WLC_GET_ANTDIV, buf, 4);
-                whd_assert_raise(ret);
+                wcm_assert_raise(ret);
```

Comment:
Here the functions are based on whd and the result check function is based on wcm. Just naming incompatibility.  Maybe later we can fix it.